thali/thalireplicationmanager.js

```diff
@@ -38,36 +39,70 @@ ThaliReplicationManager.events = {
 };
 
 /**
-* Starts the Thali replication manager with the given device name and port number
-* @param {String} deviceName the device name to advertise.
+* Starts the Thali replication manager with the given port number and db name
 * @param {Number} port the port number used for synchronization.
```

Comment:
The deviceName parameter is not being passed to the thalireplicationmanager's start function anymore. The device-name/device-identity is already available in the thalireplicationmanager.